client/src/utils/validate.ts

```diff
@@ -0,0 +1,52 @@
+import { FormErrors } from '../types/errorTypes';
+import { ContactFormData } from '../types/formDataTypes';
+
+const validateEmail = (email: string) => {
+	const emailRegex = /^([a-z\d.-]+)@([a-z\d-]+)\.([a-z]{2,8})(\.[a-z]{2,8})?$/;
+	if (!emailRegex.test(email)) {
+		return "Введите валидный email";
+	}
+	return false;
+}
+const minLength = (input: string) => {
+	const minLengthRegex = /^[a-zA-Z\d]{6,}$/;
+	if (!minLengthRegex.test(input)) {
+		return "Длина поля должна быть больше 6 символов";
+	}
+	return false;
+}
+export const validateContact = (data: ContactFormData) => {
+	const errors: FormErrors = {};
+	if (!data.name) {
+		errors.name = 'Это поле не может быть пустым'
+	}
+	if (!data.title) {
+		errors.title = 'Это поле не может быть пустым'
+	}
+	if (!data.message) {
+		errors.message = 'Это поле не может быть пустым'
+	}
+	if (!data.email) {
+		errors.email = 'Это поле не может быть пустым'
+	}
```

Comment:
Предлагаю вот так, и вынести сообщение к остальным в `content.ts`

```suggestion
    for (const [key, value] of Object.entries(data)) {
        if (!value.length) {
            errors[key] = "Это поле не может быть пустым";
        }
    }
```